src/App.tsx

```diff
@@ -1,14 +1,89 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { ChangeEvent, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
+import { FilterParams, TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+
+  const [filterParam, setFilterParam] = useState({
+    selectFilter: 'all',
+    query: '',
+  });
+
+  const [userId, setUserId] = useState(0);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>({} as Todo);
+
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Better to set `isLoading` `false` by default and to change it only before you are making API call